src/gui/qgsscrollarea.cpp

```diff
@@ -17,6 +17,7 @@
 #include <QMouseEvent>
 #include "qgsscrollarea.h"
 #include <QScrollBar>
+#include "qgslogger.h"
```

Comment:
Probably can go away?

BTW, would also be nice to re-order the includes to have "QGIS-ones" before the <Qt-ones>

Reply:
Done, thanks